docs/core/configuration.md

```diff
@@ -605,13 +605,13 @@ Here's a brief summary of the timeouts:
   on the new height (this gives us a chance to receive some more precommits,
   even though we already have +2/3)
 
-### The effect of `timeout_propose` on the proposer selection process
+### The adverse effect of using inconsistent `timeout_commit` in a network
 
-Here's an interesting question. What if the particular validator sets a very
-small `timeout_propose`?
+Here's an interesting question. What happens if a particular validator sets a
+very small `timeout_commit`, as compared to the rest of the network?
 
 Imagine there are only two validators in your network: Alice and Bob. Bob sets
-`timeout_propose` to 1s. Alice uses the default value of 3s. Bob will create
+`timeout_commit` to 1s. Alice uses the default value of 3s. Bob will create
```

Comment:
The default `timeout_commit` is 1s: https://github.com/cometbft/cometbft/blob/main/config/config.go#L1148